docarray/typing/tensor/abstract_tensor.py

```diff
@@ -117,3 +117,8 @@ def __docarray_from_native__(cls: Type[T], value: Any) -> T:
     def get_comp_backend() -> Type[AbstractComputationalBackend]:
         """The computational backend compatible with this tensor type."""
         ...
+
+    @staticmethod
+    def __docarray_normalize_value__(val: Any) -> Any:
```

Comment:
It is not clear to me what this method is intended for, even after reading the implementation for TorchTensor. Maybe we can a bit more info to the docstring here?

Reply:
sure

Reply:
this is refactor